tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -151,6 +151,31 @@ kbs_k8s_svc_host() {
 	fi
 }
 
+# Return the kbs service port number. In case ingress is configured
+# it will return "80", otherwise the pod's service port.
+#
+kbs_k8s_svc_port() {
```

Comment:
This seems to be utilized quite a lot during tests execution/library usage. Wouldn't it be better to set the port and addr once after it's deployed and use the variable instead?

Reply:
hi @ldoktor! Good point. The problem is that `kbs_k8s_svc_port()` will likely to be called on a different workflow step from the one `kbs_k8s_deploy()` and Github actions does not preserve the environment variables from one step to another. 

The way to re-export variables on subsequent steps is by writing to the `$GITHUB_ENV` file. See https://github.com/kata-containers/kata-containers/pull/9217 as an example where we faced the same problem. Writing to `$GITHUB_ENV` won't work on local outside of Github; I would need to consider that scenario too.

BTW, this problem explains why I'm using a hard-coded `$COCO_KBS_DIR`

All in all, I can figure out a way to solve that problem but I'm afraid it will make the script more complex.